lib/libspl/include/sys/uio.h

```diff
@@ -44,6 +44,8 @@
 #include <sys/types.h>
 #include_next <sys/uio.h>
 
+#include <sys/sysmacros.h>
```

Comment:
Let's keep the includes in alphaberical order and move this above `#include <sys/types.h>`.